spec/features/step_by_step_actions_spec.rb

```diff
@@ -21,11 +21,34 @@
     end
   end
 
+  context "Step by step has been submitted for 2i" do
+    before do
+      given_there_is_a_step_by_step_with_status "submitted_for_2i"
```

Comment:
I think while we only have one scenario to test, we can put this step directly in the scenario and not in a `before` block.

Reply:
Updated as suggested